File: 2_DSI/S1/Java/TP1/src/EX6/Produit.java
package EX6;

public class Produit {
    private String reference;
    private int quantite;
    private double prix;
    private double taux;

    public void InitProd(String i, int q, double p, double t) {
        reference = i;
        quantite = q;
        prix = p;
        if ((t>=0)&&(t<=100)) {
            taux = t;
        }
    }

    public void ChangerTaux(double t) {
        taux = t;
    }

    public double CalculPrix() {
        double prixTTC = prix * (1 + taux / 100);
        return prixTTC;
    }

    public void Affiche() {
        System.out.println("Référence : " + reference + ", Quantité : " + quantite +", Prix Hors Taxe : " + prix +", Taux de Taxe : " + taux + "%");
    }
}

File: 1_TI/S2/Matlab/produit.m
function [c] = produit(A,B)

    for i=1:3
        for j=1:3
            c(i,j)=0;
            for k=1:3
                c(i,j)=c(i,j)+ A(i,k)*B(k,j);
            end            
        end
    end
end
